tests/process011.hs

```diff
@@ -14,23 +14,15 @@ main = do
 
   -- shell kills itself with SIGINT,
   -- delegation off, exit code (death by signal) reported as normal
-  do let script = intercalate " "
-                    [ "exec python3 2>/dev/null"
-                    , "-c"
-                    , "'import os; os.kill(os.getpid(), 2)'"
-                    ]
+  do let script = "./process011_c"
```

Comment:
This should be `exec ./process011_c` to avoid the original issue of pdksh mangling the exit code. Alternatively an API which skips shell entirely should be used.